modulemd/v2/tests/test-modulemd-merger.c

```diff
@@ -82,6 +82,7 @@ merger_test_deduplicate (CommonMmdTestFixture *fixture,
   /* Create a ModuleIndexMerger and add both copies to it */
   merger = modulemd_module_index_merger_new ();
   g_assert_nonnull (merger);
+  g_assert_null (error);
```

Comment:
This check probably belongs on line 79, not here. Error cannot be set by `modulemd_module_index_merger_new()`.

Reply:
I shall make the required changes.